### PR TITLE
add database benchmark

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
* Find the limit of events per second that a postgres database can
handle.

```
# report.txt

Batch insert of 1,000
concurrency,inserts_per_second
1, 8116.638
2, 14160.448
3, 20323.381999999998
4, 23171.894
5, 27382.370000000003
6, 28360.913
7, 31273.359
8, 26579.333
16, 24559.041
32, 13697.913

Batch COPY FROM of 1,000
concurrency,inserts_per_second
1, 13580.28
2, 23264.471
3, 29748.702999999998
4, 34747.132
5, 39804.155999999995
6, 36720.458999999995
7, 36566.207
8, 33729.222
16, 20602.072
32, 15859.992999999999
```

### CODE REVIEW

File: awxbench/awxbench.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+- hosts: database
+  environment:
+    PATH: "/opt/rh/rh-postgresql10/root/usr/bin/:{{ ansible_env.PATH }}"
+    LD_LIBRARY_PATH: "/opt/rh/rh-postgresql10/root/usr/lib64"
+  tasks:
+    - name: Disable SELinux
+      selinux:
+        state: disabled
```
Comment (shanemcd) [inline awxbench/awxbench.yml:8]: Yikes. Why is this needed?

Comment (chrismeyersfsu) [reply awxbench/awxbench.yml:8]: I need to figure out the proper `selinux` "things" to set on the `.csv` file that I upload so that postgres can access it via the `COPY FROM` statement. I was reading the selinux coloring book. I think I know how, just ran out of time.

Comment (shanemcd) [reply awxbench/awxbench.yml:8]: Any chance running `restorecon` on those files fixes it?
